### DISCUSSION TITLE
Citekey with Short Title No Longer Working

### DISCUSSION THREAD

**Discussion by darrylmy** [retorquere/zotero-better-bibtex#2982] [General]
I just updated from the beta to Zotero 7.0.5 and Better BibTex 6.7.227, and the citekey is no longer using the Short Title for the citekey, but rather, the Title. I have confirmed this by changing both fields and the Citekey changes when the Title field is changed.

Here is what I have for the citekey generator which was, but is no longer working:

authEtAl.nopunct + year + shorttitle(2,2)

Can this be looked into for a fix?

**Comment by retorquere** [comment]
Open an issue, send a debug log.
